binding/HarfBuzzSharp.Shared/DelegateProxies.unicode.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace HarfBuzzSharp
+{
+	// public delegates
+
+	public delegate UnicodeCombiningClass CombiningClassDelegate (UnicodeFunctions ufuncs, uint unicode);
+
+	public delegate UnicodeGeneralCategory GeneralCategoryDelegate (UnicodeFunctions ufuncs, uint unicode);
+
+	public delegate uint MirroringDelegate (UnicodeFunctions ufuncs, uint unicode);
+
+	public delegate Script ScriptDelegate (UnicodeFunctions ufuncs, uint unicode);
+
+	public delegate bool ComposeDelegate (UnicodeFunctions ufuncs, uint a, uint b, out uint ab);
+
+	public delegate bool DecomposeDelegate (UnicodeFunctions ufuncs, uint ab, out uint a, out uint b);
+
+	// internal proxy delegates
+
+	[UnmanagedFunctionPointer (CallingConvention.Cdecl)]
+	public delegate UnicodeCombiningClass hb_unicode_combining_class_func_t (IntPtr ufuncs, uint unicode, IntPtr context);
```

Comment:
Make these internal so they aren't used accidentally by the users.

Reply:
Oops 👍